src/schema/external_identifiers.yaml

```
@@ -10,10 +10,19 @@ imports:
   - attribute_values # in cycle with basic_classes
 
 prefixes:
+  bioproject: 'https://identifiers.org/bioproject:'
```

Should we use https://bioregistry.io/registry/bioproject instead of identifiers.org since most of our other prefixes are using bioregistry?

```suggestion
  bioproject: 'https://bioregistry.io/bioproject:'
```

Especially since the biosample prefix right below is also using bioregistry?